jsonschema/google/events/cloud/audit/v1/LogEntryData.json

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
I'm not sure if there's a better way, but when reviewing process/automation changes, these generated output changes tend to become easily ignored noise.

Is it intended that these diffs be reviewed as well? If so it might be useful to deep link or embed some signature diff sections in the PR description.

Reply:
I wrote a detailed description for the PR, covering the `oneOf` and `enum` changes described in this file.

I intend for the review to look at the diff and approve if there are no requested changes. If the PR is not clear, please request for more information in a comment or we can also meet and sync.